smoke-tests-otel-starter/build.gradle.kts

```diff
@@ -59,4 +59,9 @@ graalvmNative {
     buildArgs.add("--initialize-at-build-time=org.junit.platform.launcher.core.LauncherConfig")
     buildArgs.add("--initialize-at-build-time=org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter")
   }
+
+  tasks.test {
+    useJUnitPlatform()
+    setForkEvery(1)
```

Comment:
I have tested with @DirtiesContext without success (telemetry missing for one test). Perhaps this topic could be revisited later with the help of #8962 and #8963.